### PR TITLE
Disable sentinel web filters.

### PR DESCRIPTION
**Pull request by AbelSara**
### Describe what this PR does / why we need it
The web filters should be disabled if current application is an API gateway.

### Does this pull request fix one issue?
#821 

### Describe how you did it
Adding configuration item `spring.cloud.sentinel.filter.enabled=false` by `EnvironmentPostProcessor`
in module `sentinel-gateway`.
### Describe how to verify it
None

### CODE REVIEW

File: spring-cloud-alibaba-sentinel-gateway/src/main/java/com/alibaba/cloud/sentinel/gateway/GatewayEnvironmentPostProcessor.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ */
+public class GatewayEnvironmentPostProcessor implements EnvironmentPostProcessor {
+    private final static String SENTINEL_FILTER_ENABLED = "spring.cloud.sentinel.filter.enabled";
+    private final static String FILTERS_DISABLED = "filters_disabled";
+
+    @Override
+    public void postProcessEnvironment(ConfigurableEnvironment environment, SpringApplication springApplication) {
+        Properties properties = new Properties();
+        properties.setProperty(SENTINEL_FILTER_ENABLED, "false");
+        PropertiesPropertySource propertySource = new PropertiesPropertySource(FILTERS_DISABLED, properties);
```
Comment (fangjian0423) [inline spring-cloud-alibaba-sentinel-gateway/src/main/java/com/alibaba/cloud/sentinel/gateway/GatewayEnvironmentPostProcessor.java]: i think it is no need to create a new `PropertiesPropertySource`, refer https://github.com/alibaba/spring-cloud-alibaba/blob/master/spring-cloud-starter-alibaba/spring-cloud-starter-bus-rocketmq/src/main/java/com/alibaba/cloud/bus/rocketmq/env/RocketMQBusEnvironmentPostProcessor.java

Comment (AbelSara) [reply spring-cloud-alibaba-sentinel-gateway/src/main/java/com/alibaba/cloud/sentinel/gateway/GatewayEnvironmentPostProcessor.java]: Thanks. I'll modify it soon.

File: spring-cloud-alibaba-sentinel-gateway/src/main/java/com/alibaba/cloud/sentinel/gateway/GatewayEnvironmentPostProcessor.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
+package com.alibaba.cloud.sentinel.gateway;
```
Comment (fangjian0423) [inline spring-cloud-alibaba-sentinel-gateway/src/main/java/com/alibaba/cloud/sentinel/gateway/GatewayEnvironmentPostProcessor.java]: add license

File: spring-cloud-alibaba-sentinel-gateway/src/main/java/com/alibaba/cloud/sentinel/gateway/GatewayEnvironmentPostProcessor.java [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.env.EnvironmentPostProcessor;
+import org.springframework.core.env.*;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Created by zhh on 2019/8/17.
```
Comment (fangjian0423) [inline spring-cloud-alibaba-sentinel-gateway/src/main/java/com/alibaba/cloud/sentinel/gateway/GatewayEnvironmentPostProcessor.java]: the author format is not correct.

Comment (AbelSara) [reply spring-cloud-alibaba-sentinel-gateway/src/main/java/com/alibaba/cloud/sentinel/gateway/GatewayEnvironmentPostProcessor.java]: Sorry. I have added the license and author.

**Pull request merged**